web/modules/custom/hello_user/src/Controller/HelloUser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\hello_user\Controller;
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Class HelloUser
```

Comment:
Please check class files from core/ to see how the function docblocks are written.